app/assets/javascripts/discourse/app/components/group-post.js

```diff
@@ -41,4 +42,9 @@ export default Component.extend({
   userUrl(username) {
     return userPath(username.toLowerCase());
   },
+
+  @discourseComputed("post.topic.title", "post.post_number")
+  titleAriaLabel(title, postNumber) {
+    return `${title} - ${I18n.t("groups.aria_post_number", { postNumber })}`;
```

Comment:
I think it might be better to add the topic title to the new i18n string, so that the order "[title] - post [number]" works for RTL langs

Reply:
Yes, good point, string is now updated.